common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1436,7 +1435,7 @@ object CelebornConf extends Logging {
       .version("0.2.0")
       .doc("Application heartbeat timeout.")
       .timeConf(TimeUnit.MILLISECONDS)
-      .createWithDefaultString("120s")
+      .createWithDefaultString("600s")
```

Comment:
This may cause some delay in cleaning up application resource, shall we consider support application self request celeborn master release all resources using rpc?

Reply:
How about 240s? 120s is so long....

Reply:
It highly depends on the scale of the cluster, 600s may be too large for a small cluster, should be good to use the spark default heartbeat timeout value.

Reply:
Changed to 300s cc @RexXiong @pan3793

Reply:
> Changed to 300s cc @RexXiong @pan3793

300s seems more reasonable.